Makefile

```diff
@@ -122,16 +116,17 @@ clean: info down
 	@for i in $(DIRS); do if [ -d "web/$$i" ]; then echo "Removing web/$$i..."; docker run --rm -v $(shell pwd):/mnt $(IMAGE_PHP) sh -c "rm -rf /mnt/web/$$i"; fi; done
 
 net:
-ifeq ($(shell docker network ls -q -f Name=$(COMPOSE_NET_NAME)),)
+ifeq ($(strip $(shell docker network ls | grep $(COMPOSE_NET_NAME))),)
+ifneq ($(COMPOSE_SUBNET_RANGE),)
```

Comment:
no reason in extra var

the logic here is simple
1) if net exists (`docker network ls` returns it) & no `IPRANGE` in `.env` then "read subnet from existing net and put to .IPRANGE"
2) if there's no net & no range defined in .env - call `docker network create $(COMPOSE_NET_NAME)` to make docker allocate new net for use & then  "read subnet from existing net and put to .IPRANGE"